docs/ota-client-guide/modules/ROOT/assets/attachments/aktualizr-monitoring-zabbix-template.xml

```diff
@@ -0,0 +1,309 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<zabbix_export>
+    <version>4.0</version>
+    <date>2020-02-13T10:13:43Z</date>
+    <groups>
+        <group>
+            <name>Templates/Applications</name>
+        </group>
+    </groups>
+    <templates>
+        <template>
+            <template>Aktualizr Client</template>
+            <name>Aktualizr Client</name>
+            <description>Aktualizr client resources consumption monitoring.</description>
+            <groups>
+                <group>
+                    <name>Templates/Applications</name>
+                </group>
+            </groups>
+            <applications>
+                <application>
+                    <name>Aktualizr/Application</name>
+                </application>
+            </applications>
+            <items>
+                <item>
+                    <name>aktualizr.threads.count</name>
```

Comment:
Did we lose the proc.cpu.util entries?

Reply:
I've removed this key, but I can revert it'll take a couple of mins!

Reply:
currently, template contain 3 keys for monitoring
```
aktualizr.rss.memory.usage - memory (aktualizr)
aktualizr.threads.count - threads (aktualizr)
vfs.dev.write.sectors -  disk usage (entire system)
```

Reply:
Ah, cool. What was `proc.cpu.util` for, then? Did you end up finding that `aktualizr.rss.memory.usage` was more useful?

Reply:
It was just an experiment with Zabbix keys,  this key stands for CPU utilization by the aktualizr, I just do not want to overload template.
Any keys which are defined in Zabbix can be added really in a couple of mins on-demand as separate PR

Reply:
Cool. Then this looks good to go. Thanks!